src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -40,74 +40,119 @@ public String sayGoodMorning(boolean isMorning) {
     // 1. What will the returned value be if I run sayGoodMorning(false)?
     // Change the returned value in the method below to your answer. It is case-sensitive.
     public String one() {
-        return "";
+        return "Good day!";
     }
 
     // 2. What will the output be if I run sayGoodMorning(true)?
     // Change the returned value in the method below to your answer. It is case-sensitive.
     public String two() {
-        return "";
+        return "Good morning!";
     }
 
     // 3. What will the output be if I run sayGoodMorning("Hello".equals("Hello"))?
     // Change the returned value in the method below to your answer. It is case-sensitive.
     public String three() {
-        return "";
+        return "Good morning!";
     }
 
     // 4. What will the output be if I run sayGoodMorning(!"A word".equals("Another word"))
     public String four() {
-        return "";
+        return "Good morning!";
     }
 
     // 5. What will the output be if I run sayGoodMorning(25 != 25)
     public String five() {
-        return "";
+        return "Good day!";
     }
 
     // 6. Use a conditional statement to return "Correct!" if the input is more than 7
     // or "Wrong!" if not
     public String six(int num) {
-        return "Not implemented yet!";
+        String out="";
+        if (num>7){
+            out="Correct!";
+        }else {
+            out="Wrong!";
+        }
+        return out;
     }
 
     // 7. Use a conditional statement to return "Correct!" if the input is false
     // or "Wrong!" if not
     public String seven(boolean bool) {
-        return "Not implemented yet!";
+        String out="";
+        if (!bool){
+            out="Correct!";
+        }else {
+            out="Wrong!";
+        }
+        return out;
     }
 
     // 8. Use a conditional statement to return "Correct!" if numOne is more than or equal to numTwo
     // or "Wrong!" if not
     public String eight(int numOne, int numTwo) {
-        return "Not implemented yet!";
+        String out="";
+        if (numOne>=numTwo){
+            out="Correct!";
+        }else{
+            out="Wrong!";
+        }
+        return out;
     }
 
     // 9. Use a conditional statement to return true if the array provided is not empty
     // or false if it is empty
     public boolean nine(int[] nums) {
-        return false;
+        boolean out;
+        if (!(nums.length==0)){
```

Comment:
You could also have just said either `(nums.length != 0)` or `(nums.length>0)` which might involve slightly less brain power when you want to understand what the code is doing.